examples/examples_nodejs_test.go

```diff
@@ -103,8 +103,8 @@ func TestAccTable(t *testing.T) {
 func TestAccServicebusMigration(t *testing.T) {
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
-			Dir:           filepath.Join(getCwd(t), "servicebus-migration-test"),
-			RunUpdateTest: true,
+			Dir: filepath.Join(getCwd(t), "servicebus-migration-test"),
+			// RunUpdateTest: true,
```

Comment:
Did you mean to uncomment this?

Reply:
Ah, this was where I found out that our previous "UpdateTest" was installing the older SDK, but then due to the provider binary configuration, was still using the local dev version of the provider ... which broke things while upgrading and was also an ineffective test. We then doubled down on improving our snapshot testing instead.

> - Disable "RunUpdateTest" in nodejs as this can't be used with the LocalProviders configuration as it will always use the local provider binary even when using the old SDK. This option is essentially broken and needs replacing with proper upgrade tests.